src/statmodels.jl

```diff
@@ -159,44 +159,70 @@ df_residual(obj::RegressionModel) = error("df_residual is not defined for $(type
 
 ## Nms are the coefficient names, corresponding to rows in the table
 type CoefTable
-    mat::Matrix
+    cols::Tuple
     colnms::Vector
     rownms::Vector
-    pvalcol::Integer
-    function CoefTable(mat::Matrix,colnms::Vector,rownms::Vector,pvalcol::Int=0)
-        nr,nc = size(mat)
-        0 <= pvalcol <= nc || error("pvalcol = $pvalcol should be in 0,...,$nc]")
+    function CoefTable(cols::Tuple,colnms::Vector,rownms::Vector)
+        nc = length(cols)
+        nrs = map(length,cols)
+        nr = max(nrs...)
         length(colnms) in [0,nc] || error("colnms should have length 0 or $nc")
         length(rownms) in [0,nr] || error("rownms should have length 0 or $nr")
-        new(mat,colnms,rownms,pvalcol)
+        all(unique(nrs) .== nr) || error("Elements of cols should have equal lengths")
```

Comment:
I don't think you need to call `unique` here. Also, no need to take the `max`, just use the first length. Finally, it would be nice to add `", but got $nrs"` at the end of the error message.